packages/opentelemetry-node/src/NodeTracerProvider.ts

```diff
@@ -39,7 +39,34 @@ export class NodeTracerProvider extends BasicTracerProvider {
     super(config);
 
     this._pluginLoader = new PluginLoader(this, this.logger);
-    this._pluginLoader.load(config.plugins || DEFAULT_INSTRUMENTATION_PLUGINS);
+
+    /**
+     * For user supplied config of plugin(s) that are loaded by default,
+     * merge the user supplied and default configs of said plugin(s)
+     */
+    let mergedUserSuppliedPlugins: Plugins = {};
+
+    for (const pluginName in config.plugins) {
+      if (DEFAULT_INSTRUMENTATION_PLUGINS.hasOwnProperty(pluginName)) {
+        mergedUserSuppliedPlugins[pluginName] = {
+          ...DEFAULT_INSTRUMENTATION_PLUGINS[pluginName],
+          ...config.plugins[pluginName],
+        };
+      } else {
+        mergedUserSuppliedPlugins[pluginName] = config.plugins[pluginName];
+        // enable user-supplied plugins unless explicitly disabled
+        if (mergedUserSuppliedPlugins[pluginName].enabled === undefined) {
```

Comment:
Is `enabled` a required plugin config? If not, I think it would be a good field to add.

Reply:
By not requiring it, we would be able to enable by default. Is it preferred to require `enabled` be explicitly set, or default to `enabled: true` when not set?

Reply:
Just to clarify my intent: that `enabled` is a plugin config parameter, but if left unset, it would default to true. So if someone brings their plugin `horse` and specifies the path, it will automatically enable without them having to add `enabled: true` explicitly.

Reply:
Seems like it is not required:

https://github.com/open-telemetry/opentelemetry-js/blob/b1fe2c81b9842dca18dba7db7d87c15fc95820b6/packages/opentelemetry-api/src/trace/instrumentation/Plugin.ts#L56-L60

But it also seems to default to `true` but I'm not sure why/how that gets lost, and why I am defaulting it to true for any user supplied plugin.